tests/lib/rules/no-async-actions.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-async-actions.js');
+const RuleTester = require('eslint').RuleTester;
+
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const message = 'Do not use async actions';
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  }
+});
+
+ruleTester.run('no-async-actions', rule, {
+  valid: [
```

Comment:
We need a valid example to ensure this rule does not trigger on `actions` in an irrelevant object/class. It should only trigger on the `actions` key inside an Ember controller/component/route.

Reply:
^